curriculum/challenges/english/25-front-end-development/quiz-css-grid/66ed8fedf45ce3ece4053eb3.md

```diff
@@ -149,45 +149,45 @@ What is the shorthand for the `column-gap` and `row-gap` properties?
 
 #### --text--
 
-Which property defines on which row line the item will start?
+What is the difference between an implicit and explicit grid?
 
 #### --distractors--
 
-`row-start`
+Implicit grids use the `grid-template-columns` property while explicit grids use the `grid-template-rows` property.
 
 ---
 
-`grid-row-begin`
+Explicit grids use the `grid-template-columns` property while implicit grids use the `grid-template-rows` property.
 
 ---
 
-`grid-row`
+Implicit grids use the `grid-template-columns` or `grid-template-rows` properties to create columns while rows and columns are automatically created in explicit grids.
 
 #### --answer--
 
-`grid-row-start`
+Explicit grids use the `grid-template-columns` or `grid-template-rows` properties to create columns while rows and columns are automatically created in implicit grids.
 
 ### --question--
 
 #### --text--
 
-Which property automatically allocates rows based on a fixed size?
+Which of the following units represents a fraction of the space within the grid container?
```

Comment:
```suggestion
Which of the following units represent a fraction of the space within the grid container?
```

Reply:
Isn't that right? It's singular

Reply:
"units" is plural so it's represents.

Reply:
Which uses the third-person singular form

Reply:
Sorry I meant to say represents (hasty me).

Reply:
I will resolve this. Because I am not sure.